### ISSUE TITLE
Domains in signup: can't use keyboard to select a domain choice (accessibility blocker)

### ISSUE THREAD

**Issue by lancewillett** [Automattic/wp-calypso#19194]
_Moved here from WordPress core Trac: https://core.trac.wordpress.org/ticket/42344_

> When creating new WordPress.com site, and it is time to choose the address of your site, you can write the name of the site. At that point it gives you options for the addresses. These addresses can not be accessed with tab key. As it is not possible to choose the address of the Wordpress site with the tab key, the site is not accessible for the blind.

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/start/
2. At https://wordpress.com/start/domains enter a possibly domain name
3. Enter some text
4. See the available choices
5. Try to use your keyboard to navigate to your choice, and choose it

#### What I expected
To be able to hit "tab" key and move through the domain choices.

#### What happened instead
I wasn't able to use the keyboard to select a domain choice.

#### Screenshot / Video
![domain-keyboard](https://user-images.githubusercontent.com/66797/32072470-80f76d16-ba47-11e7-803f-09f6f2b176f8.gif)

**Comment by lancewillett** [comment]
Fixed with 86a7f8a 🎉
